src/bust.py

```diff
@@ -71,6 +74,7 @@ def __init__(
 
         self._finished: bool = False
         self._playing_index: Optional[int] = None
+        self.temp_audio_file: str = f"temp_audio.ogg"
```

Comment:
Busty is technically multiserver (untested lol) and having a single temp audio file name could cause conflicts between multiple busts happening on different servers. I think with this implementation the temp file will also be dumped into the central busty repo folder instead of being contained. See code linked below for where and how we save attachment files, and maybe make the temp audio file something like `attachments/<guild_id>/temp_audio.ogg` instead

https://github.com/anoadragon453/busty/blob/main/src/discord_utils.py#L37-L40